tmd/io/conversion.py

```diff
@@ -17,11 +17,15 @@ def convert_morphio_soma(morphio_soma):
         tmd_soma (Soma)
     """
     points = morphio_soma.points
+    try:
+        d = morphio_soma.diameters
+    except AttributeError:
+        d = np.zeros(points.shape[0], dtype=points.dtype)
```

Comment:
This should be removed as well as it is trying to fall back when a neurom soma is passed.